controllers/controllers/workloads/cfprocess_controller.go

```diff
@@ -109,6 +109,21 @@ func (r *CFProcessReconciler) ReconcileResource(ctx context.Context, cfProcess *
 	return ctrl.Result{}, nil
 }
 
+func needsAppWorkload(cfApp *korifiv1alpha1.CFApp, cfProcess *korifiv1alpha1.CFProcess) bool {
+	if cfApp.Spec.DesiredState != korifiv1alpha1.StartedState {
+		return false
+	}
+	return desiredProcessInstances(cfProcess) > 0
+}
+
+func desiredProcessInstances(cfProcess *korifiv1alpha1.CFProcess) int {
+	if cfProcess.Spec.DesiredInstances == nil {
+		return 0
```

Comment:
I wonder if  `0` is a reasonable default, since the app does not do anything when there are 0 instances. If a kubectl user wants to push an app without specifying too much stuff wouldn't `1` be a better default? Scaling down to zero sounds like a special case rather than a default.